### PR TITLE
Fix #406

### LINKED ISSUE #406: _gfx_driver_packages not defined when choosing sway option (current master build)
VirtualBox_Arch_30_04_2021_23_01_50

### PR DESCRIPTION
**Pull request by dylanmtaylor**
Closes #406 

🚨 PR Guidelines:

# New features *(v2.2.0)*

All future work towards *`v2.2.0`* is done against `master` now.<br>
Any patch work to existing verions will have to create a new branch against the tagged versions.

# Describe your PR

If the changes has been discussed in an Issue, please tag it so we can backtrace from the Issue later on.<br>
If the PR is larger than ~20 lines, please describe it here unless described in an issue.

# Testing

Any new feature or stability improvement should be tested if possible.
Please follow the test instructions at the bottom of the README or use the ISO built on each PR.

### CODE REVIEW

**Comment by dylanmtaylor** [comment]
Seems to work from some quick testing

File: profiles/sway.py [hunk 1]
```diff
@@ -13,13 +13,13 @@ def _prep_function(*args, **kwargs):
 	other code in this stage. So it's a safe way to ask the user
 	for more input before any other installer steps start.
 	"""
-	if "nvidia" in _gfx_driver_packages:
+	__builtins__['_gfx_driver_packages'] = archinstall.select_driver()
```
Comment (Torxed) [inline profiles/sway.py:16]: This driver will never be installed unless `xorg` is also installed. Which I can't see that sway does?

**Review by Torxed — CHANGES_REQUESTED** [review]

File: profiles/sway.py (hunk 1 above)
Comment (dylanmtaylor) [reply profiles/sway.py:16]: Wayland uses hardware acceleration as far as I know. Graphics drivers provide that and I believe the same ones are used for Wayland and X

**Comment by dylanmtaylor** [comment]
Need to test that before merging it.

**Comment by dylanmtaylor** [comment]
Tested to work.
VirtualBox_Arch_01_05_2021_09_55_50

**Pull request merged**